### DISCUSSION TITLE
using blackboxexporter

### DISCUSSION THREAD

**Discussion by jamessewell** [carlosedp/cluster-monitoring#124] [Q&A] [answered]
Hi,

I've followed the blackbox exporter instructions - but the components seem strangely absent from this repo?

It's mentioned in the Readme - but nowhere else?

Is this possible?

Cheers,
James

**Comment by jamessewell** [comment]
I'm really confused about this. You have a container in your docker.io repo for blackbox, but seemingly not the infra to support it.

Could it be that it never worked or was intended to work?

**Comment by minecraftchest1** [comment]
I was also wondering how to get back box exporter to work.

**Comment by carlosedp** [comment, marked as answer]
The blackbox exporter is not used in the stack, you need to provide the configuration for your own use-case.
